src/Extension/FluentExtension.php

```diff
@@ -23,18 +23,28 @@
 /**
  * Basic fluent extension
  *
+ * When determining whether a field is localised, the following config options are checked in order:
+ * - translate (uninherited, for each class in the chain)
+ * - field_exclude
+ * - field_include
+ * - type_exclude
+ * - type_include
```

Comment:
I believe this should be called `data_exclude` and `data_include` to be consistent with the rest of the code.

Reply:
Yep - seems like a typo. I've pushed a commit that fixes that.